tests/opt-freeze-coords/input.dat

```diff
@@ -6,6 +6,8 @@ O   1.000000   1.000000   1.000000
 H   2.000000   1.000000   1.000000
 H   1.000000   2.000000   1.000000
 units angstrom
+no_com
+no_reorient
```

Comment:
Didn't we want this test making sure that the user could offer any old molecule and then the driver would be smart enough to set no_com/reorient when it noticed the frozen coordinates?